src/siema.js

```diff
@@ -305,6 +319,7 @@ export default class Siema {
     this.selector.removeEventListener('mouseup', this.mouseupHandler);
     this.selector.removeEventListener('mouseleave', this.mouseleaveHandler);
     this.selector.removeEventListener('mousemove', this.mousemoveHandler);
+    document.removeEventListener('keydown', this.keydownHandler);
```

Comment:
It makes more sense @znk. It should be controlled from app level. At least optinal, but enabling thing like this out of the box may harm a lot.